### ISSUE TITLE
[4.0] global menu options

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#34248]
### Step 1
Go to the global options for a menu item and set a value for all 4 options
![chrome_2021-05-27_18-32-30](https://user-images.githubusercontent.com/1296369/119875499-b914bc00-bf1e-11eb-9df5-48b3e524c3bc.png)

### Step 2
Create a new menu item and check the settings in the page display tab - expected behaviour would be to see the values from step 1

![chrome_2021-05-27_18-34-03](https://user-images.githubusercontent.com/1296369/119875638-e5303d00-bf1e-11eb-9130-6f4fc3d99e73.png)

###  Step 3
Now we know that the page heading and page class global settings are ignored - what about the browser title that is at least displayed in the menu item options. Nope thats ignored

![chrome_2021-05-27_18-38-32](https://user-images.githubusercontent.com/1296369/119875731-0002b180-bf1f-11eb-9d26-666e748c5782.png)

### Summary
The only one of the four options that actually does anything is the third one "Show Page Heading" but even that doesnt really doing anything as it is meaningless without a value for  "Page Heading"

### Conclusion
There is zero point in having any of these global options for a menu item as they dont work as their is no code to use them and they can be removed - _**unless I have missed something**_

**Comment by tonypartridge** [comment]
They used to work in J3, so I’d expect them to work in J4... we have actually be using them for meta management and page H1’s. 

And page class to define custom classes within the theme... 

Wouldn’t it make more sense to add the support rather than remove it?

**Comment by brianteeman** [comment]
are you sure they work in j3? The broken behaviour seems identical to me.

**Comment by chmst** [comment]
They don't work in J3 either, at least in content. @tonypartridge magine that all sites have the same title? a11y and seo ....

**Comment by chmst** [comment]
Finally found what happens and where. 

Before a PR I would like to get your opinion if we can remove the site title form gobal params. Equal title for all pages is an accessibility issue if it works. @brianteeman 
 <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34248">issues.joomla.org/tracker/joomla-cms/34248</a>.</sub>

**Comment by brianteeman** [comment]
I agree equal titles is wrong. Hard to confirm that this param creates that though as I dont see where it can be used

**Comment by chmst** [comment]
It does. I will prepare a draft PR.and let the param, then you can see it. <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34248">issues.joomla.org/tracker/joomla-cms/34248</a>.</sub>

**Comment by brianteeman** [comment]
great

**Comment by richard67** [comment]
Re-opening this issue since the PR has been closed without merging.
